src/main/java/uk/gov/companieshouse/confirmationstatementapi/interceptor/validation/CompanyNumberValidationInterceptor.java

```diff
@@ -22,9 +22,6 @@
     @Value("${MAX_ID_LENGTH}")
     private int maxIdLength;
 
-    @Value("${MAX_COMPANY_NUMBER_LENGTH}")
```

Comment:
Wondering why this has gone and why we use a MAX_ID_LENGTH here? I'd have thought a company number max length is quite different to an id max length.

Reply:
It is. This is being used to truncate the string as we do not want to be logging very long strings.

Reply:
So sounds like we should not use / reference MAX_ID_LENGTH here but keep the MAX_COMPANY_NUMBER_LENGTH, use this and set to 10 in the config (to match the max length in the reg exp).

Reply:
Ok. Discussed this and now understand the motivation. Agreed to rename the 'maxIdLength' var though to something more helpful, like 'truncationLength'.